apps/passport-client/components/screens/ImportBackupScreen.tsx

```diff
@@ -0,0 +1,386 @@
+            selectedPcdIds: new Set(
+              mergeablePcds
+                .filter((pcd) => {
+                  const folder =
+                    parsedCollection.getFolderOfPCD(pcd.id) ?? NoFolderSymbol;
+                  return selectedFolders.has(folder);
+                })
+                .map((pcd) => pcd.id)
+            ),
+            folderCounts: pcdFolders
+          });
+        }
+      }
+    })();
+  }, [filesContent, importState, existingPcdCollection]);
+
+  // When the user selects or de-selects a folder for inclusion in the merge
+  const toggleFolder = useCallback(
+    (folder: string | symbol) => {
+      if (importState.state === "valid-file-selected") {
+        const { selectedFolders } = importState;
+        if (selectedFolders.has(folder) && selectedFolders.size > 1) {
+          selectedFolders.delete(folder);
+        } else {
+          selectedFolders.add(folder);
+        }
+        setImportState({ ...importState, selectedFolders });
+      }
+    },
+    [importState]
+  );
+
+  return (
+    <>
+      <MaybeModal />
+      <AppContainer bg="gray">
+        <ScreenNavigation label={"Home"} to="/"></ScreenNavigation>
+        <Container>
+          <Spacer h={8} />
+          <H2>Import Backup Data</H2>
+          <Spacer h={24} />
+          {importState.state === "initial" && (
+            <>
+              <p>
+                If you have previously exported a backup of your account, you
+                can restore any lost PCDs by importing the backup data.
+              </p>
+              <p>
+                Importing data will not overwrite any of your existing PCDs.
+              </p>
+              <p>
+                To begin, select a backup file by clicking the button below.
+              </p>
+              <Spacer h={8} />
+              <Button onClick={() => openFilePicker()}>Select file</Button>
+              <Spacer h={8} />
+            </>
+          )}
+          {importState.state === "valid-file-selected" && (
+            <>
+              {importState.mergeablePcdIds.size == 0 && (
+                <>
+                  <p>
+                    The selected file does not contain any new PCDs. You may try
+                    to restore from another backup.
+                  </p>
+                  <Spacer h={8} />
+                  <Button onClick={() => openFilePicker()}>Select file</Button>
+                  <Spacer h={8} />
+                </>
+              )}
+              {importState.mergeablePcdIds.size > 0 && (
+                <>
+                  <p>
+                    The selected file contains{" "}
+                    <strong>{importState.mergeablePcdIds.size}</strong> new
+                    PCDs.
+                  </p>
+                  <div>
+                    Import PCDs from the following backed-up folders:
+                    <Folders>
+                      {[
+                        [NoFolderSymbol, 1] as [symbol, number],
```

Comment:
I don't understand what's being done here.  The generation of folderCounts above seems to already include a count for the root folder, so what does an extra count of 1 accomplish here?